vendor-extra/LiberoPageBundle/src/EventListener/MainListener.php

```diff
@@ -24,11 +24,20 @@ public function __construct(EventDispatcherInterface $dispatcher)
 
     public function onCreatePage(CreatePageEvent $event) : void
     {
+        switch ($event->getRequest()->attributes->get('libero_page')['type'] ?? null) {
+            case 'homepage':
```

Comment:
special case doesn't feel very extension, but if the alternative is some sort of complex configuration that maps `homepage` to `listing-grid`...

Reply:
Did wonder about that, but not sure how exchangeable they are. The template itself can be overridden, which is probably enough for now.